index.html

```diff
@@ -942,17 +937,16 @@ <h6 class="title">FOSSASIA Links</h6>
 			</footer>
 		</div>
 		
-				
-	<script src="js/jquery.min.js"></script>
-        <script src="js/bootstrap.min.js"></script>
-        <script src="js/flexslider.min.js"></script>
-        <script src="js/lightbox.min.js"></script>
-        <script src="js/masonry.min.js"></script>
-        <script src="js/flickr.js"></script>
-        <script src="js/twitterfetcher.min.js"></script>
-        <script src="js/spectragram.min.js"></script>
-        <script src="js/smooth-scroll.min.js"></script>
-        <script src="js/parallax.js"></script>
-        <script src="js/scripts.js"></script>
```

Comment:
Why done this?

Reply:
i read somewhere that this specifies the MIME type of the linked file.

https://www.tutorialspoint.com/html/html_script_tag.htm

Reply:
No, I meant you deleted the same lines and re-added it.
And, this dismantled some line spacing. Fix them :+1:

Reply:
oh sure. i'll be closing this pr and making a new one then 👍

Reply:
No. Just make a commit. And then squash commits :wink:

Reply:
i face a lot of issues in squashing. 
so, i'm closing this